### PR TITLE
Fix `mongo` configuration

### PR DESCRIPTION
**Pull request by b1ron**

### CODE REVIEW

**Review by AlekSi — APPROVED** [review]

File: tests/mongo.yml [hunk 1]
```diff
@@ -96,9 +92,6 @@ results:
       # https://jira.mongodb.org/browse/SERVER-14832
       - mongo/jstests/core/query/distinct/distinct_array1.js
 
-      # https://github.com/FerretDB/FerretDB/issues/2236
-      - mongo/jstests/core/query/distinct/distinct1.js
-
       # https://github.com/FerretDB/FerretDB/issues/2231
```
Comment (AlekSi) [inline tests/mongo.yml:95]: Hm, that one is closed. Please check

Comment (b1ron) [reply tests/mongo.yml:95]: Seems to be fixed in FerretDB/FerretDB/pull/2658. It also passes locally.

Comment (AlekSi) [reply tests/mongo.yml:95]: So should we remove it from here?

Comment (b1ron) [reply tests/mongo.yml:95]: Yes I believe so, it's fixed so I removed it from fail. Same deal with auth2.js. It was fixed in FerretDB/FerretDB/pull/2726 and it passes locally.

Comment (AlekSi) [reply tests/mongo.yml:95]: Oops! Let's remove them in a new PR

Comment (b1ron) [reply tests/mongo.yml:95]: Okie

**Pull request merged**